src/cloudlabs/views_host.py

```diff
@@ -34,9 +35,7 @@ def add():
     form.admin_ssh_key.choices = [(key.id, key.label)
                                   for key in g.user.ssh_keys]
     if form.validate_on_submit():
-        fields = {
-            'user_id': g.user.id,
-        }
+        fields = {'user_id': g.user.id, }
```

Comment:
We could get rid of the trailing comma here actually. (I originally thought there would be more entries defined here, but then there weren't!)